### PR TITLE
Feature/login page

### PR DESCRIPTION
**Pull request by JCourtney22**

### CODE REVIEW

File: src/containers/Main/style.css [hunk 1]
```diff
@@ -2,4 +2,10 @@
     width: 100%;
     height: 100%;
     overflow: auto;
+    background-image: url('../../assets/images/Book\ pages.jpg');
```
Comment (JonHarrison) [inline src/containers/Main/style.css:5]: not sure if this would be better with %20 rather than \

File: src/pages/Login/style.css [hunk 2]
```diff
@@ -1,3 +1,12 @@
 .App-login {
     width: 400px;
+    background-color: rgba(69, 156, 209, 0.5);
```
Comment (JonHarrison) [inline src/pages/Login/style.css:3]: would be better in a var

Comment (JCourtney22) [reply src/pages/Login/style.css:3]: do you still want me to do this?

**Review by JonHarrison — APPROVED** [review]
approved with comments

**Pull request merged**